src/components/Modal/index.tsx

```diff
@@ -167,7 +171,7 @@ export default function Modal({
                         hidden={true}
                         minHeight={minHeight}
                         maxHeight={maxHeight}
-                        mobile={isMobile}
+                        mobile={isMobile ?? undefined}
```

Comment:
why is this ever undefined?

Reply:
I did this because warning was firing when isMobile was false, but was weird because it should be expecting a boolean - it may have to do with the way teh styled components is passing props down ill take another look

Reply:
If you do a pull, I touched the same thing in this file to address the same problem earlier, did it slightly differently though by just not passing through the mobile prop